pkg/v1/partial/uncompressed.go

```diff
@@ -148,7 +153,13 @@ func (i *uncompressedImageExtender) Manifest() (*v1.Manifest, error) {
 		},
 	}
 
-	ls, err := i.Layers()
+	// TODO this shouldn't depend on layers being populated
```

Comment:
We don't pass options into manifest generation, so we do some hackery here. Calling `i.Layers()` is slow, so this saved 10s of seconds.

Reply:
Why is `Layers` slow?

Reply:
I imagine it was calculating digests for the first time (since we called `CompressedLayers` above but I'm not 100% sure.